docs/vm-images.md

```diff
@@ -166,45 +206,53 @@ a description of our support for Windows images.
 
 <sup><b>*</b></sup> Chrome Remote desktop does not support Ubuntu 20.04, but it does support Ubuntu 22.04.
 
-## Other Images
+### Other Images
 
 The HPC Toolkit strives to provide flexibility wherever possible. It is possible
 to set a VM image in many HPC Toolkit modules. While we do not officially
 support images not listed here, other public and custom images should work with
 the majority of modules with or without further customization, such as custom
 startup-scripts.
 
-## Slurm on GCP Custom Images
+### Slurm on GCP
+
+For Slurm images, please see [SchedMD's GitHub repository](https://github.com/SchedMD/slurm-gcp/blob/5.7.6/docs/images.md#public-image).
+
+#### Publically Published Images
+
+SchedMD publishes "Slurm on GCP" public images, which have the following [documentation][slurm-gcp-images]. From there, you can see which public images are available and which software is installed on them.
```

Comment:
I think you need to draft text here that is crystal clear that the Terraform module version and Packer module version must be the same. That should be the first sentence. The second (and third?) sentences should be that SchedMD publishes these images and that our modules will enforce compatibility when using images published by SchedMD.

The next paragaph should cover custom images. We can review that text in next round.